### PR TITLE
docs: update swagger path to full url

### PR DESCRIPTION
**Pull request by jyeshe**

### CODE REVIEW

**Comment by jyeshe** [comment]
Maybe changed to a service url with the swagger yaml as a param

File: README.md [hunk 1]
```diff
@@ -225,10 +225,11 @@ GET /status                              - returns middleware status
 
 ## OpenAPI specs
 
-Under the `/swagger` path there are two files providing a swagger specificiation of the endpoints for both api versions:
+There are two files providing a swagger specificiation of the endpoints for respective api versions:
+
+- https://mainnet.aeternity.io/mdw/swagger/swagger_v1.yaml
+- https://mainnet.aeternity.io/mdw/swagger/swagger_v2.yaml
```
Comment (thepiwo) [inline README.md:231]: thats incorrect

File: README.md [hunk 2]
```diff
@@ -225,10 +225,11 @@ GET /status                              - returns middleware status
 
 ## OpenAPI specs
 
-Under the `/swagger` path there are two files providing a swagger specificiation of the endpoints for both api versions:
+There are two files providing a swagger specificiation of the endpoints for respective api versions:
+
+- https://mainnet.aeternity.io/mdw/swagger/swagger_v1.yaml
```
Comment (thepiwo) [inline README.md:230]: can we add the links so they open in a hosted (e.g. petstore) swagger ui?

File: README.md (hunk 1 above)
Comment (jyeshe) [reply README.md:231]: What is incorrect? Please try with .art for v2.

File: README.md (hunk 2 above)
Comment (jyeshe) [reply README.md:230]: That was my first comment on the PR (: swagger editor imports the file, will try replacing the pet store.

File: README.md (hunk 1 above)
Comment (jyeshe) [reply README.md:231]: It is not deplyed. Please try with .art for v2.

File: README.md (hunk 2 above)
Comment (jyeshe) [reply README.md:230]: > can we add the links so they open in a hosted (e.g. petstore) swagger ui?

The guys from swagger UI had thought about it (: It requires a form request controlled by javascript. The same for other examples I have found.

Comment (jyeshe) [reply README.md:230]: > can we add the links so they open in a hosted (e.g. petstore) swagger ui?

The guys from swagger UI had thought about it (: It requires a form request controlled by javascript. The same for other examples I have found.

Comment (thepiwo) [reply README.md:230]: you can add `?url=` to the domain

Comment (jyeshe) [reply README.md]: was falling on the validator, thanks!

**Review by thepiwo — APPROVED** [review]
:tada:

**Pull request merged**